DigitalEcosystem/refined/metal_classifier/multiple_bins/multi_bin_classifier.py

```diff
@@ -0,0 +1,286 @@
+#!/usr/bin/env python
```

Comment:
The code looks OK - should we be storing the `png` files in git LFS?

Reply:
Probably a good idea - I'll go ahead and track those